packages/cactus-plugin-ledger-connector-iroha/docs/architecture/run-transaction-endpoint-transact-cactuskeychainref.puml

```diff
@@ -0,0 +1,33 @@
+@startuml
```

Comment:
@hxlaf Please delete the `./packages/cactus-plugin-ledger-connector-iroha/docs/` (in general everything that's not applicable for Iroha that came from the copying operation in the beginning needs to go)

Reply:
Thank you! Done.
That makes sense.

> @hxlaf Please delete the `./packages/cactus-plugin-ledger-connector-iroha/docs/` (in general everything that's not applicable for Iroha that came from the copying operation in the beginning needs to go)